qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -304,10 +304,14 @@ def run(
 
         schedules, num_memory_slots_list = _to_schedule_list(run_input, backend=backend)
 
-        # get the acquires instructions and simulation times
-        t_span, measurement_subsystems_list, memory_slot_indices_list = _get_acquire_data(
-            schedules, backend.options.subsystem_labels
-        )
+        # get the acquires sample times and subsystem measurement information
+        (
+            acquire_time_list,
+            measurement_subsystems_list,
+            memory_slot_indices_list,
+        ) = _get_acquire_data(schedules, backend.options.subsystem_labels)
+
+        t_span = [[0.0, x * self.options.solver._dt] for x in acquire_time_list]
```

Comment:
non-blocking: consider the rename `_get_acquire_instruction_timings`, and consider passing `self.options.solver._dt` so that the output of the helper is more generic.

Reply:
I've implemented both suggestions and added a test directly verifying that correct `t_span` values are being returned by the newly named `_get_acquire_instruction_timings`.